pos_payment_terms/__openerp__.py

```diff
@@ -0,0 +1,38 @@
+###############################################################################
+#     Point Of Sale - Payment Terms for POS Odoo
+#    Copyright (C) 2016 KMEE INFORMATICA LTDA (http://www.kmee.com.br)
```

Comment:
@lfdivino 
Please use small copyright like this one: https://github.com/OCA/bank-payment/blob/8.0/account_banking_payment_export/__openerp__.py#L2